### PR TITLE
feat: request reviewer on automatic backport

### PR DESCRIPTION
**Pull request by codebytere**
If the author of a PR has sufficient permissions (i.e write access, we should request their review on the automatically backported pull request.

cc @MarshallOfSound

### CODE REVIEW

File: src/Probot.d.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export interface PullRequest {
   head: Ref;
   base: Ref;
   merged: boolean;
-  author: GitHubUser;
+  user: GitHubUser;
```
Comment (codebytere) [inline src/Probot.d.ts]: per [documentation](https://developer.github.com/v3/pulls/#get-a-single-pull-request), this should be `user` not `author`

File: src/utils.ts [hunk 2]
```diff
@@ -32,6 +32,18 @@ export const labelMergedPR = async (context: Context, pr: PullRequest, targetBra
   }
 };
 
+const checkUserIsGovMember = async (context: Context, user: String) => {
```
Comment (MarshallOfSound) [inline src/utils.ts]: We should use https://developer.github.com/v3/repos/collaborators/#review-a-users-permission-level to keep this generic for other folks using trop

Comment (codebytere) [reply src/utils.ts]: ooh great find, didn't see that one!

will do

File: src/utils.ts [hunk 3]
```diff
@@ -189,13 +198,22 @@ export const backportImpl = async (robot: Application,
...
           base: targetBranch,
           title: pr.title,
           body: createBackportComment(pr),
           maintainer_can_modify: false,
-        }))).data;
+        })));
+
+        // If user has sufficient permissions (i.e has write access)
+        // request their review on the automatically backported pull request
+        if (checkUserHasWriteAccess(context, pr.user.login)) {
```
Comment (MarshallOfSound) [inline src/utils.ts]:
```suggestion
        if (await checkUserHasWriteAccess(context, pr.user.login)) {
```

Comment (codebytere) [reply src/utils.ts]: derp

**Pull request merged**